src/main/java/com/coherent/training/api/kapitsa/base/RestAssuredClient.java

```diff
@@ -0,0 +1,55 @@
+package com.coherent.training.api.kapitsa.base;
+
+import io.restassured.response.Response;
+
+import java.io.File;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public class RestAssuredClient {
+    public Response get(String url, Map<String, String> headerMap){
+        return given().headers(headerMap)
+                .when().get(url);
+    }
+
+    public Response get(String url, Map<String, String> headerMap, Map<String, String> param){
+        return given().headers(headerMap)
+                .queryParams(param)
+                .when().get(url);
+    }
+
+    public Response post(String url, Map<String, String> headerMap, Map<String, String> formMap){
+        return given().headers(headerMap)
+                .formParams(formMap)
+                .when().post(url);
+    }
+
+    public <T> Response post(String url, Map<String, String> headerMap, T bodyClass){
+        return given().headers(headerMap)
+                .body(bodyClass)
+                .when().post(url);
+    }
+
+    public Response post(String url, Map<String, String> headerMap, File file){
```

Comment:
let's make post with all needed params and then overload it with the params needed (by call base post (with non-null empty bodies if needed))